packages/grid/_modules_/grid/components/toolbar/GridFilterToolbarButton.tsx

```diff
@@ -11,9 +13,21 @@ import {
 import { gridPreferencePanelStateSelector } from '../../hooks/features/preferencesPanel/gridPreferencePanelSelector';
 import { GridPreferencePanelsValue } from '../../hooks/features/preferencesPanel/gridPreferencePanelsValue';
 import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridTranslationKeys } from '../../models/api/gridLocaleTextApi';
 import { GridApiContext } from '../GridApiContext';
 
+const useStyles = makeStyles(
+  (theme) => ({
+    list: {
+      margin: theme.spacing(1),
```

Comment:
How about?

```suggestion
      margin: theme.spacing(1, 1, 0.5),
```

**Before**

<img width="152" alt="Screenshot 2021-04-12 at 01 17 03" src="https://user-images.githubusercontent.com/3165635/114324893-2619f100-9b2d-11eb-95ae-098af41bfc83.png">

**After**

<img width="133" alt="Screenshot 2021-04-12 at 01 19 50" src="https://user-images.githubusercontent.com/3165635/114324897-329e4980-9b2d-11eb-9b4b-75258b776764.png">